app/assets/javascripts/client/resources/InventoryParticipant.js

```diff
@@ -0,0 +1,7 @@
+PDRClient.factory('InventoryParticipant', ['$resource', 'UrlService', function($resource, UrlService) {
```

Comment:
This resource isn't using the IIFE style, and it's relying on the PDRClient variable.